newslet/website/admin.py

```diff
@@ -1,4 +1,8 @@
 from django.contrib import admin
-from .models import GuestSurvey
+from .models import TestModel, WisdomNote, NewsTags, NewsCategory, Formularz
```

Comment:
Tego TestModel, WisdomeNote, NewsTags, NewsCategory - juz nie ma na masterze, bo je wywalilem. Takze mysle, ze wziales stara wersje kodu, z niej zrobiles branch i teraz wraz z Twoimi zmianami moze sie pojawic stary kod. Moja propozycja - zapisz to co zrobiles, wez w nowym oknie sciagnij z mastera ostatni kod i tam dodaj swoje zmiany z tym Formularzem.

Reply:
zgadzam sie z Michalem, wyglada na to ze wprowadzales zmiany na starej wersji.